src/Snt/Capi/Repository/Article/FindParameters.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Snt\Capi\Repository\Article;
+
+final class FindParameters
+{
+    /**
+     * @var string
+     */
+    private $publicationId;
+
+    /**
+     * @var int
+     */
+    private $articleId;
+
+    /**
+     * @param string $publicationId
+     * @param int $articleId
+     */
+    public function __construct($publicationId, $articleId)
+    {
+        $this->publicationId = $publicationId;
+        $this->articleId = $articleId;
+    }
```

Comment:
moreover I think that those two classes can be merged into one, we can discuss it tomorrow